org.eclipse.wildwebdeveloper/language-servers/package.json

```diff
@@ -12,6 +12,7 @@
     "vscode-json-languageserver": "./vscode-json-languageserver-1.0.1.tgz",
     "typescript": "2.8.3",
     "typescript-language-server": "0.3.7",
-    "node-debug2": "./node-debug2-1.25.5.tgz"
+    "node-debug2": "./node-debug2-1.25.5.tgz",
+    "yaml-language-server": "0.2.1"
```

Comment:
Why not 0.3.2 ?

Reply:
@yanxi0830 Your answer here is important as we cannot start the IP approval for yaml-ls before we're sure of which version to take (IP approvals are version specific).

Reply:
I was following the version used in https://github.com/redhat-developer/vscode-yaml, which was just updated from 0.2.1 to 0.3.2 a couple of days ago, so 0.3.2 would be the version to take.